tools/java/org/unicode/cldr/util/CLDRPaths.java

```diff
@@ -48,6 +48,8 @@
     public static final String ANNOTATIONS_DERIVED_DIRECTORY = CldrUtility.getPath(CLDRPaths.COMMON_DIRECTORY, ANNOTATIONS_DERIVED_SUBDIR);
     public static final String VALIDITY_DIRECTORY = CldrUtility.getPath(CLDRPaths.COMMON_DIRECTORY, VALIDITY_SUBDIR);
 
+    public static final String STAGING_DIRECTORY = CldrUtility.getPath(CLDRPaths.BASE_DIRECTORY, "../cldr-staging/");
```

Comment:
I don't think we want to have GenerateProductionData write directly into the staging directory. The current documented ICU workflow has its output going into the cldr-aux directory; only when that data is determined to be save-able to we copy to staging. Many times (most times>) that GenerateProductionData is run, it will not be to generated archivable production data.